src/components/SignInButton/index.tsx

```diff
@@ -1,17 +1,21 @@
 import React, { FC } from 'react';
-import { style } from './style';
-import { StatusBar, Text, TouchableOpacity, View } from 'react-native';
+import { Text, TouchableOpacity } from 'react-native';
 import { SignInButtonPropsType } from './type';
+import { style } from './style';
 
 export const SignInButton: FC<SignInButtonPropsType> = props => {
-  const { googleSignIn } = props;
+  const { signIn } = props;
+
+  const textStyle = () => {
```

Comment:
make it like a variable, not function. and apply cond rendering

Reply:
i fix it